config/routes.rb

```diff
@@ -134,7 +134,7 @@ def external_redirect(path_prefix, target)
     end
     get "/consultations/:consultation_id/:id" => "_#_", as: "consultation_html_attachment"
     get "/consultations/:consultation_id/outcome/:id" => "_#_", as: "consultation_outcome_html_attachment"
-    get "/consultations/:consultation_id/public-feedback/:id" => "_#_", as: "consultation_public_feedback_html_attachment"
+    get "/consultations/:consultation_id/public-feedback/:content_id" => "_#_", as: "consultation_public_feedback_html_attachment"
```

Comment:
These changes all look good on the surface (without knowing too much about all the different HTML attachment types!). But I suspect we'll want to keep the old routes for the time being, so that existing HTML attachment URLs continue to work. We can then remove the old routes in https://trello.com/c/3BeDlugo/2761-republish-whitehall-documents-containing-non-english-html-attachments-5 (please add to that card as an acceptance criteria once this PR is merged!)